supriya/tools/ugentools/VOsc3.py

```diff
@@ -4,11 +4,12 @@
 
 class VOsc3(PureUGen):
     """
+    A wavetable lookup oscillator which can be swept smoothly across wavetables.
 
     ::
 
         >>> vosc_3 = ugentools.VOsc3.ar(
-        ...     bufpos=bufpos,
```

Comment:
Could rename `bufpos` to `buffer_id`. That will make this more consistent with other UGens that use buffers.

Please also rename `freq_1`, `freq_2` and `freq_3` to `frequency_*`.

One of the things I'm trying to do in this project (as I get around to it) is clarifying the names that we derive from SuperCollider. I'm trying to avoid weird abbreviations and use complete words whenever naming things. This isn't always perfect, but it's my goal.